src/applications/disability-benefits/2346/schemas/address-schema.js

```diff
@@ -361,6 +361,19 @@ export const addressUISchema = (
         hideIf: () => !isMilitaryBaseAddress,
         hideOnReviewIfFalse: true,
         useDlWrap: true,
+        updateSchema: (formData, schema, uiSchema) => {
+          const countryFormData = get(path, formData);
+          const countryNameNormalized = countryFormData.toUpperCase();
+          if (
```

Comment:
I don't think checking for what they send us in this manner is efficient and doesn't capture what we're trying to do.  

The goal is to look at what they're sending us, see if its something our inputs can handle without going into edit mode and then if it is, we don't go into edit mode

```
country && listOfGoodCountries.includes(country)
```
would be more appropriate than having all of these individual conditions.  Also I don't think they would send us a string of 4 country codes as a single country.